src/main/java/com/faforever/client/preferences/Preferences.java

```diff
@@ -18,7 +18,7 @@
   private final WindowPrefs mainWindow;
   private final ForgedAlliancePrefs forgedAlliance;
   private final LoginPrefs login;
-  private final ChatPrefs chat;
+  private final ChatPrefs chatPrefs;
```

Comment:
if we change this to `chatPrefs`, we'd also have to change the others as well.
However, the reason why I haven't called it `*Prefs` is that these files get serialized to `client.prefs`, so it will look like this:

    "clientPrefs": {
      ...
    }

and I prefered:

    "client": {
      ...
    }

We can discuss this but we just need to keep it consistent, also whenever we change it, we break the user's preferences.